[receiver/windowsperfcounters] Flaky Test - Test_WindowsPerfCounterScraper

Observed here: https://github.com/open-telemetry/opentelemetry-collector-contrib/runs/7008709942?check_suite_focus=true#step:7:2750

```
=== RUN   Test_WindowsPerfCounterScraper/Standard
    windowsperfcounters_scraper_test.go:184: 
        	Error Trace:	windowsperfcounters_scraper_test.go:184
        	Error:      	Received unexpected error:
        	            	A counter with a negative denominator value was detected.
        	Test:       	Test_WindowsPerfCounterScraper/Standard
=== RUN   Test_WindowsPerfCounterScraper/SumMetric
=== RUN   Test_WindowsPerfCounterScraper/NoMetricDefinition
=== RUN   Test_WindowsPerfCounterScraper/InvalidCounter
--- FAIL: Test_WindowsPerfCounterScraper (0.25s)
    --- FAIL: Test_WindowsPerfCounterScraper/Standard (0.25s)
```

--- a/.chloggen/winperf-retry-rollover.yaml
+++ b/.chloggen/winperf-retry-rollover.yaml
@@ -1,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: windowsperfcountersreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Retry counter retrieval once, when error indicates possible rollover
+
+# One or more tracking issues related to the change
+issues: [14343]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:

--- a/pkg/winperfcounters/watcher.go
+++ b/pkg/winperfcounters/watcher.go
@@ -19,6 +19,7 @@ package winperfcounters // import "github.com/open-telemetry/opentelemetry-colle
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/winperfcounters/internal/third_party/telegraf/win_perf_counters"
 )
@@ -104,9 +105,19 @@ func (pc *perfCounter) Path() string {
 }
 
 func (pc *perfCounter) ScrapeData() ([]CounterValue, error) {
-	err := pc.query.CollectData()
-	if err != nil {
-		return nil, fmt.Errorf("failed to collect data for performance counter '%s': %w", pc.path, err)
+	if err := pc.query.CollectData(); err != nil {
+		pdhErr, ok := err.(*win_perf_counters.PdhError)
+		if !ok || pdhErr.ErrorCode != win_perf_counters.PDH_CALC_NEGATIVE_DENOMINATOR {
+			return nil, fmt.Errorf("failed to collect data for performance counter '%s': %w", pc.path, err)
+		}
+
+		// A counter rolled over, so the value is invalid
+		// See https://support.microfocus.com/kb/doc.php?id=7010545
+		// Wait one second and retry once
+		time.Sleep(time.Second)
+		if retryErr := pc.query.CollectData(); retryErr != nil {
+			return nil, fmt.Errorf("failed retry for performance counter '%s': %w", pc.path, err)
+		}
 	}
 
 	vals, err := pc.query.GetFormattedCounterArrayDouble(pc.handle)

--- a/receiver/windowsperfcountersreceiver/windowsperfcounters_scraper_test.go
+++ b/receiver/windowsperfcountersreceiver/windowsperfcounters_scraper_test.go
@@ -65,7 +65,6 @@ func mockPerfCounterFactory(mpc mockPerfCounter) newWatcherFunc {
 }
 
 func Test_WindowsPerfCounterScraper(t *testing.T) {
-	t.Skip("Flaky Test - See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11445")
 	type testCase struct {
 		name string
 		cfg  *Config

